forum/views/search.py

```diff
@@ -0,0 +1,151 @@
+"""
+Search API Views
+"""
+
+from typing import Any, Dict, List, Optional, Tuple
```

Comment:
lets use `dict` instead of `Dict`

Reply:
consequently we can remove `Dict` import

Reply:
Can we do this in a separate PR? I'll create another PR for this.